actionview/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 ## Rails 8.0.0.beta1 (September 26, 2024) ##
 
+*   Add a configuration `csp_meta_tag_nonce_attribute` to allow renaming the csp_meta_tag helper nonce attribute name.
```

Comment:
Can you move this above `8.0.0.beta1` since this PR can't be included in something that's already been released?